lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp

```diff
@@ -1857,56 +1861,68 @@ void LifetimeChecker::updateInstructionForInitState(DIMemoryUse &Use) {
   assert(isa<StoreInst>(Inst) && "Unknown store instruction!");
 }
 
+void LifetimeChecker::processUninitializedReleaseOfBox(
+    AllocBoxInst *ABI, SILInstruction *Release, bool consumed,
+    SILBasicBlock::iterator InsertPt) {
+  assert(ABI == Release->getOperand(0));
+  SILBuilderWithScope B(Release);
+  B.setInsertionPoint(InsertPt);
```

Comment:
Here you're updating the insertion point but not the debug scope. Is this intentional?

Reply:
I just followed the model in other parts of the file. If this is wrong then the other places are wrong as well.